libs/database/src/lib/entities/index.ts

```diff
@@ -1,38 +1,37 @@
 export * from "./application.entity";
 export * from "./delayed-job.entity";
-export * from "./demographic.entity";
 export * from "./demographic-entry.entity";
+export * from "./demographic.entity";
 export * from "./form-submission.entity";
-export * from "./framework.entity";
 export * from "./framework-user.entity";
+export * from "./framework.entity";
```

Comment:
Not that it's a big deal, but why these changes and the the others here around files that aren't part of this PR? `framework` should come before `framework-user` in an alphabetical sort